Add align to rustc-attrs unstable book (#76430)
rust-lang/rust: src/doc/unstable-book/src/language-features/rustc-attrs.md

with `cargo check`) as an alternative to `rustc -Z print-type-sizes`
that is way more verbose.

Options provided by `#[rustc_layout(...)]` are `debug`, `size`, `align`,

> `abi`. Note that it only work best with sized type without generics.

Reviewer: ```suggestion
`abi`. Note that it only works on sized types without generics.
```

Reviewer: At least, I couldn't get it to work on generics. :)
